.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+# Cut a release whenever a new tag is pushed to the repo.
+name: Release
+on:
+  # Can be triggered from the tag.yaml workflow
+  workflow_call:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+    secrets:
+      BCR_PUBLISH_TOKEN:
```

Comment:
Is this a new secret, or something that already likely exists?

What does it need to be? Is this a GitHub token or something else?

Reply:
we need to create a Personal Access Token for the credentials it will use to open the PR.

see "Create a Personal Access Token" under https://github.com/bazel-contrib/publish-to-bcr?tab=readme-ov-file#setup